linode/manager
# change: [M3-7721] - Finish `AGLB` to `ACLB` rename

## Description 📝

This PR finishes migrating AGLB (Akamai Global Load Balancer) to ACLB (Akamai Cloud Load Balancer)

> [!important]
> This PR will cause Cloud Manager to make fetches to `/v4beta/aclb` instead of `/v4beta/aglb`. This should be merged once the API switches over

## How to test 🧪

- The API team must rename the endpoint on their end for this change to work in the dev environment
- For now, just turn on the MSW and verify http://localhost:3000/loadbalancers/ is working as expected. Calls should be made to `/v4beta/aclb` instead of `/v4beta/aglb`
- Look through the diff and verify I did a good job at finding and replacing all instanced of `aglb` with `aclb`

## As an Author I have considered 🤔
